ivadomed/main.py

```diff
@@ -72,15 +72,15 @@ def get_parser():
 
 
 def create_path_model(context, model_params, ds_train, path_output, train_onehotencoder):
-    path_model = os.path.join(path_output, context["model_name"])
-    if not os.path.isdir(path_model):
+    path_model = Path(path_output, context["model_name"])
```

Comment:
best to use `get()` pattern and potentially convert that string to utilize the `keyword.py`

Reply:
keyword.py changes will be in another pr